packages/gatsby/cache-dir/api-ssr-docs.js

```diff
@@ -145,7 +145,10 @@ exports.onPreRenderHTML = true
  * This is useful for setting wrapper component around pages that won't get
```

Comment:
```suggestion
 * This is useful for setting a wrapper component around pages that won't get
```

If possible, it would be awesome to improve the readability while we're in here.